homeassistant/components/sense/__init__.py

```diff
@@ -91,11 +92,19 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     except SENSE_EXCEPTIONS as err:
         raise ConfigEntryNotReady(str(err) or "Error during realtime update") from err
 
+    async def update_trend():
```

Comment:
```suggestion
    async def _async_update_trend():
```

Since Home Assistant has both async and sync code in it we usually prefix async function with `async`